guides/common/modules/proc_configuring-isc-dhcp.adoc

```diff
@@ -0,0 +1,51 @@
+[id="Configuring_isc_dhcp_{context}"]
```

Comment:
This should be covered by the installer and the existing guides. I don't think we should document manual installation in the primary guide.

I've been wondering if we should document the reference bits on https://docs.theforeman.org (similar to how today it's on theforeman.org) or rather in [smart-proxy.git](https://github.com/theforeman/smart-proxy). The latter makes sense to me because it's more focused on developers. You can easily apply the same in a development setup.

Reply:
Removed as duplicate.